src/tools/idlc/src/types.c

```diff
@@ -108,7 +108,7 @@ generate_implicit_sequences(
   memset(&visitor, 0, sizeof(visitor));
   visitor.visit = IDL_MEMBER | IDL_SEQUENCE;
```

Comment:
This also needs `IDL_CASE`, and in `emit_implicit_sequence` the assert should also accept a case (line 49)

Reply:
Added them, probably got lost when I removed the debug code. Thanks!